test/mini_mime_test.rb

```diff
@@ -10,6 +10,12 @@ def test_that_it_has_a_version_number
     refute_nil ::MiniMime::VERSION
   end
 
+  def test_configure
+    MiniMime.configure do |config|
```

Comment:
what is this api for? its a bit confusing

Reply:
See https://thoughtbot.com/blog/mygem-configure-block. Its a configuration pattern I see a lot in ruby gems ([1](https://github.com/carrierwaveuploader/carrierwave#configuring-carrierwave), [2](https://docs.bugsnag.com/platforms/ruby/rails/configuration-options/), [3](https://github.com/doorkeeper-gem/doorkeeper#api-mode)). While it hides the detail of referencing `MiniMime::Configuration`, it's not exactly necessary. You could just do `MiniMime::Configuration.var = value` instead. Shall I 🔥 it?